modules/nw-sriov-hwol-about-hardware-offloading.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-hardware-offloading.adoc
+
+:_content-type: CONCEPT
+[id="about-hardware-offloading_{context}"]
+= About hardware offloading
+
+Hardware offloading is a method for processing data that removes data processing tasks from the CPU and transfers, or "offloads", them to the dedicated data processing unit (DPU) of a network interface controller.
```

Comment:
`"offloads",` the `,` is needed?

Reply:
Agreed, that's not quite right. Will have a think and take this up with peer reviewer. Thanks!